HPE3PARPSToolkit-CLI.psm1

```diff
@@ -34380,11 +34379,2491 @@ Function Stop-3parWsapi()
+ )
+
+ Write-DebugLog "Start: In Move-3parDomain - validating input values" $Debug 
+ #check if connection object contents are null/empty
+ if(!$SANConnection)
+ {
+	#check if connection object contents are null/empty
+	$Validate1 = Test-ConnectionObject $SANConnection
+	if($Validate1 -eq "Failed")
+	{
+		#check if global connection object contents are null/empty
+		$Validate2 = Test-ConnectionObject $global:SANConnection
+		if($Validate2 -eq "Failed")
+		{
+			Write-DebugLog "Connection object is null/empty or Connection object UserName,password,IPAaddress are null/empty. Create a valid connection object using New-SANConnection" " ERR: "
+			Write-DebugLog "Stop: Exiting Move-3parDomain since SAN connection object values are null/empty" $Debug 
+			Return "FAILURE : Exiting Move-3parDomain since SAN connection object values are null/empty"
+		}
+	}
+ }
+
+ $plinkresult = Test-PARCli -SANConnection $SANConnection
+ if($plinkresult -match "FAILURE :")
+ {
+   write-debuglog "$plinkresult"
+   Return $plinkresult
+ }
+
+ $Cmd = " movetodomain "
+
+ if($Vv)
+ {
+	$Cmd += " -vv "
+ }
+
+ if($Cpg)
+ {
+	$Cmd += " -cpg "
+ }
+
+ if($Host)
+ {
+	$Cmd += " -host "
+ }
+
+ if($F)
+ {
+	$Cmd += " -f "
+ }
+	
+ if($ObjName)
+ {
+	$Cmd += " $ObjName "
+ }
+ 
+ if($DomainName)
+ {
+	$Cmd += " $DomainName "
+ }
+
+ $Result = Invoke-3parCLICmd -Connection $SANConnection -cmds  $Cmd
+ Write-DebugLog "Executing Function : Move-3parDomain Command -->" INFO: 
+ 
+ if($Result -match "Id")
+ {
+	$Cnt = $Result.count
+		
+ 	$tempFile = [IO.Path]::GetTempFileName()
+	$LastItem = $Result.Count -1  
+	
+	foreach ($s in  $Result[0..$LastItem] )
+	{		
+		$s= [regex]::Replace($s,"^ ","")			
+		$s= [regex]::Replace($s," +",",")	
+		$s= [regex]::Replace($s,"-","")
+		$s= $s.Trim()
+		Add-Content -Path $tempfile -Value $s				
+	}
+	Import-Csv $tempFile 
+	del $tempFile 	
+ }
+ 
+ if($Result -match "Id")
+ {
+	return  " SUCCESS : EXECUTING Move-3parDomain"
+ }
+ else
+ {			
+	return "FAILURE : While Executing Move-3parDomain `n $Result"
+ }
+ 
+} ##  End-of Move-3parDomain
+
+##########################################################################
+######################### FUNCTION New-3parDomain #########################
+##########################################################################
+Function New-3parDomain()
+{
+<#
+  .SYNOPSIS
+   New-3parDomain : Create a domain.
+
+  .DESCRIPTION
+   The New-3parDomain command creates system domains.
+
+  .EXAMPLE
+	New-3parDomain -Domain_name xxx
+  
+  .EXAMPLE
+	New-3parDomain -Domain_name xxx -Comment "Hello"
+
+  .PARAMETER Domain_name
+	Specifies the name of the domain you are creating. The domain name can be no more than 31 characters. The name "all" is reserved.
+	
+  .PARAMETER Comment
+   Specify any comments or additional information for the domain. The comment can be up to 511 characters long. Unprintable characters are not allowed. 
+   The comment must be placed inside quotation marks if it ontains spaces.
```

Comment:
Correct the sentance " The comment must be placed inside quotation marks if it **ontains** spaces."

Reply:
DONE.